mechanics/RELEASE-VERSIONING-PRINCIPLES.md

```diff
@@ -0,0 +1,727 @@
+# Knative Release Principles
```

Comment:
Add missing frontmatter (required for rendering in knative.dev - the title here gets rendered as the H1 (if the traditional `# Title` exists, the site renders an unwanted duplicate))
```suggestion
---
title: "Knative release principles"
linkTitle: "Release principles"
weight: 50
type: "docs"
---
```

Reply:
Done.